press/press/doctype/app_release/app_release.py

```diff
@@ -505,8 +491,8 @@ def check_python_syntax(dirpath: str) -> str:
 	- -q: quiet, only print errors (stdout)
 	- -o: optimize level, 0 is no optimization
 	"""
-
-	command = f"python3 -m compileall -q -o 0 {dirpath}"
+	python_exec = _get_python_executable()
```

Comment:
`sys.executable` should suffice if you are trying to fetch the interpreter for Press' current runtime. This is assuming `get_bench_path` returns the path of the bench Press is installed on.

> Warning: I don't have a Press setup; was just browsing this...for fun 😄